diff --git a/pyproject.toml b/pyproject.toml
index ed6b952..e705d08 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -37,6 +37,7 @@ dependencies = [
     "ffmpeg-python>=0.2.0",
     "humanfriendly>=10.0",
     "pillow>=11.3.0",
+  "xdg-base-dirs>=6.0.2",
 ]
 
 [project.scripts]
diff --git a/salmon/uploader/spectrals.py b/salmon/uploader/spectrals.py
index 072dc37..1c8781c 100644
--- a/salmon/uploader/spectrals.py
+++ b/salmon/uploader/spectrals.py
@@ -9,6 +9,7 @@ import shutil
 import subprocess
 import time
 from os.path import dirname, join
+from xdg_base_dirs import xdg_data_home
 
 import click
 import oxipng
@@ -292,7 +293,7 @@ def _open_specs_in_windows(spectrals_path):
 
 async def _open_specs_in_web_server(specs_path, all_spectral_ids):
     spectrals.set_active_spectrals(all_spectral_ids)
-    symlink_path = join(dirname(dirname(__file__)), "web", "static", "specs")
+    symlink_path = join(f"{xdg_data_home()}/smoked-salmon", "web", "static", "specs")
 
     shutdown = True
     try:
diff --git a/salmon/web/__init__.py b/salmon/web/__init__.py
index b531969..475e5ab 100644
--- a/salmon/web/__init__.py
+++ b/salmon/web/__init__.py
@@ -6,6 +6,7 @@ import aiohttp_jinja2
 import click
 import jinja2
 from aiohttp_jinja2 import render_template
+from xdg_base_dirs import xdg_data_home
 
 from salmon import cfg
 from salmon.common import commandgroup
@@ -28,14 +29,14 @@ def web():
 def create_app():
     app = aiohttp.web.Application()
     add_routes(app)
-    aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader(join(dirname(__file__), "templates")))
+    aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader(join(f"{xdg_data_home()}/smoked-salmon/web", "templates")))
     return loop.run_until_complete(loop.create_server(app.make_handler(), host="0.0.0.0", port=web_cfg.port))
 
 
 async def create_app_async():
     app = aiohttp.web.Application()
     add_routes(app)
-    aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader(join(dirname(__file__), "templates")))
+    aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader(join(f"{xdg_data_home()}/smoked-salmon/web", "templates")))
     runner = aiohttp.web.AppRunner(app)
     await runner.setup()
     site = aiohttp.web.TCPSite(runner, "0.0.0.0", web_cfg.port)
@@ -47,7 +48,7 @@ async def create_app_async():
 
 
 def add_routes(app):
-    app.router.add_static("/static", join(dirname(__file__), "static"), follow_symlinks=True)
+    app.router.add_static("/static", join(f"{xdg_data_home()}/smoked-salmon/web", "static"), follow_symlinks=True)
     app.router.add_route("GET", "/", handle_index)
     app.router.add_route("GET", "/spectrals", spectrals.handle_spectrals)
     app["static_root_url"] = web_cfg.static_root_url
