diff --git a/build.gradle.kts b/build.gradle.kts
index ae0ed21e..56a57077 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -165,7 +165,6 @@ tasks {
         destinationDirectory = devOutputDir
     }
     shadowJar {
-        from(collectReachabilityMetadata)
         archiveBaseName = project.name
         archiveClassifier = ""
         archiveVersion = ""
diff --git a/gradle.properties b/gradle.properties
index 7f7f7773..c921b9e7 100644
--- a/gradle.properties
+++ b/gradle.properties
@@ -1,4 +1,3 @@
 org.gradle.jvmargs=-Xmx4G
 org.gradle.caching=true
 org.gradle.console=colored
-org.gradle.configuration-cache=true
